project.php

```diff
@@ -761,16 +761,6 @@ function do_project_info_table()
     {
         $comments = $project->comments;
 
-        // Automatically prepend second round comments template for Beginners Only
-        // The r2 in the template name is an artifact of the old two-round structure
-        if ($project->difficulty == "beginner")
-        {
```

Comment:
How will this happen now?

Reply:
We will have this code as a PROD hack (or something much like it):
```php

        // Automatically prepend second round comments template for Beginners Only
        // The r2 in the template name is an artifact of the old two-round structure
        if ($project->difficulty == "beginner")
        {
            if ($round && $round->is_a_mentor_round() )
            {
                $comments = "
                <p>This book has changed from a BEGINNERS ONLY project to a MENTORS project.</p>

                <p><b>NOTE:</b> You may find a list of which beginner has done which page, and  the number of pages proofread by each, at the <a href='https://www.pgdp.net/c/tools/proofers/for_mentors.php'>For Mentors page</a>.</p>

                <p>Also, should you be issued a page that shows any of the Project Facilitators as the P1 proofreader, you may safely skip the feedback--unless, of course, we NEED it! That page will have been reclaimed, then proofread by one of us in order to move the project on to P2.</p>
                " . $comments;
            }
        }
```

Reply:
I was actually looking at that bit of code today and thought to myself -- "Aaah, that's where it would go!". One other comment to make about what is currently BGr2 -- because all BEGINs are held in P3 Waiting until after the newest part has been out of P2 for at least 100 days, and merged before being released into P3, you can probably not worry about removing that bit of comments after P2. Only the destination part for the merge will need to have PCs updated, and the squirrel doing the merge can as easily remove/comment out that as the rest of the BEGIN PCs, while putting in the new PCs for P3+.

Reply:
> you can probably not worry about removing that bit of comments after P2

The PROD hack won't actually alter the PCs, it is only added when rendered, so there is nothing for the PM to remove.